src/loaders/workers.js

```diff
@@ -15,6 +15,24 @@ export default ({ queue }) => {
   const workers = new ConnectionLoader(
     async ({ provisionerId, workerType, options, filter }) => {
       const raw = await queue.listWorkers(provisionerId, workerType, options);
+
+      raw.workers = await Promise.all(
```

Comment:
While this technically works, it bypasses a lot of the logic we have surrounding loading statuses.

What you want to do instead is override the resolver on a `LatestTask` to return a `TaskStatus` at that property:

```js
// in https://github.com/taskcluster/taskcluster-web-server/blob/master/src/resolvers/Workers.js
LatestTask: {
  status(parent, args, { loaders }) {
    return loaders.status.load(parent.taskId);
  }
}
```

If instead of getting the status you want the actual run, rename to `run`, and pull the run out (since I don't think there is a TC API call for just getting a single run):

```js
LatestTask: {
  run(parent, args, { loaders }) {
    return loaders.status.load(parent.taskId).runs
      .find(run => run.runId === parent.runId);
  }
}
```

Something like that.